runtime/src/bank/tests.rs

```diff
@@ -9997,16 +9993,7 @@ fn calculate_test_fee(
     message: &SanitizedMessage,
     lamports_per_signature: u64,
     fee_structure: &FeeStructure,
-    support_set_accounts_data_size_limit_ix: bool,
 ) -> u64 {
-    let mut feature_set = FeatureSet::all_enabled();
-
-    if !support_set_accounts_data_size_limit_ix {
-        feature_set.deactivate(
-            &solana_sdk::feature_set::include_loaded_accounts_data_size_in_fee_calculation::id(),
-        );
-    }
-
```

Comment:
the feature_set is no longer used. `calculate_fee()` takes hard-coded `false` to exclude account bytes from fee calc.